digestbot/core/db/models.py

```diff
@@ -18,5 +18,5 @@ class Message:
     reply_users_count: int
     thread_length: int
     channel_id: str
-    link: str
+    link: Optional[str]
```

Comment:
or Union[str, None]

Reply:
Optional[str] is just shorthand for Union[str, None], but is more readable IMHO.